dbt/README.md

```diff
@@ -8,10 +8,10 @@ This directory stores the configuration for building our data catalog using
 ### In this document
 
 * [🖼️ Background: What does the data catalog do?](#%EF%B8%8F-background-what-does-the-data-catalog-do)
-* [🔨 How to rebuild models using GitHub Actions](#-how-to-rebuild-models-using-github-actions)
```

Comment:
It feels more natural to me to flow from "Background" > "Developing" > "Add model" > "Rebuild model" rather than "Background" > "Rebuild model" > "Developing" > "Add model", so I went ahead and reordered the sections. Happy to revert if you have a strong preference for the previous outline of the doc.